vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -117,8 +117,8 @@ if ($windowsSku -eq "2019") {
             # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
             "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01182023-095c864a",
             # NPM (Network Policy Manager). Owner: jaer-tsun (Jaeryn)
-            "mcr.microsoft.com/containernetworking/azure-npm:v1.4.29",
-            "mcr.microsoft.com/containernetworking/azure-cns:v1.4.29")
```

Comment:
Is v1.4.29 used in production? We should still keep the old versions for a while until customers upgrade their clusters to use the new version. Normally we keep [N-2, N] for package and [N-1, N] for image.

Reply:
I found that v1.4.29 is not used in the PR description.